src/components/ebay-table/component.ts

```diff
@@ -60,6 +68,19 @@ export default class EbayTable extends Marko.Component<Input, State> {
         return selected;
     }
 
+    getSortedColStateFromInput(input: Input) {
```

Comment:
So this seems to support multi column sort, what if a team wants one column sort and everything else resets when one column is sorted?

Reply:
+1 on this. In fact, DS has maintained a single sort is the default with multi-sort possible, but somewhat unlikely to be implemented. I think we should build for single-sort for now.

Reply:
Good question, we should emit single clicked column in `on-sort` event

```
{
  sorted: { col1: "none", col2: "asc", col3: "asc" }
  sortedColumn: {col3: "asc"}
}
```

So user need to store `sortedColumn` as state then do

```
<@header sort=(sortedColumn["col" + index] ?? true)>
  Col ${index}
</@header>
```

Reply:
or add sorting mode `single | multiple` and emit `sorted` obj differently. I think emitting event like above gives user more choice.

Reply:
I think we can do single column `sorted: {col3:"asc"}` in sorting event and user can still implement multiple sort if sort state properly. Will refine PR.